Gems/AtomLyIntegration/CommonFeatures/Code/Source/Mesh/MeshComponentController.cpp

```diff
@@ -196,9 +219,9 @@ namespace AZ
 
         // [GFX TODO] [ATOM-13339] Remove the ModelAsset id fix up function in MeshComponentController
         // Model id was changed due to fix for [ATOM-13312]. We can remove this code when all the levels are updated.
-        void FixUpModelAsset(Data::Asset<RPI::ModelAsset>& modelAsset)
+        void FixUpModelAsset(Data::Asset<RPI::ModelAsset>& [[maybe_unused]]modelAsset)
         {
-            Data::AssetId assetId;
+           /* Data::AssetId assetId;
```

Comment:
Please remove commented out code. Or if it's a short term thing please add a comment and create a github task to track it.

Reply:
Thanks, this function was deleted in another changelist. Forgot I had it commented out locally.